Testnet/DAOContract/DAOContract/DAOContract.cs

```diff
@@ -0,0 +1,301 @@
+﻿using Stratis.SmartContracts;
+using System;
+
+[Deploy]
+public class DAOContract : SmartContract
```

Comment:
Consider renaming to `OwnedDAOContract` or similar to reflect the fact that the funds are only intended to be added by the owner.